webapp/public/index.html

```diff
@@ -6,6 +6,17 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
+    <meta property="og:title" content="EOS RATE" />
+    <meta property="og:url" content="https://eosrate.io/" />
+    <meta property="og:type" content="article" />
+    <meta
+      property="og:description"
+      content=" Rate Block Producers - EOSIO Blockchain Ratings by EOS Costa Rica "
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/eoscostarica/eos-rate/a179f1a4e1b2b9a1d6fee398b3978640d4b19716/webapp/public/logo-mainnet.svg"
```

Comment:
this is the wrong image, lets use this one: 


![eos-rate-social-media](https://user-images.githubusercontent.com/5632966/132560243-c7218411-5e6c-45b8-b2f3-0a047abc4b0b.png)